contracts/src/CrossChainApplications/GETTING_STARTED.md

```diff
@@ -4,21 +4,57 @@ This section walks through how to build an example cross-chain application on to
 
 ## Step 1: Create Initial Contract
 
-Create a new file called `MyExampleCrossChainMessenger.sol` in the directory that will hold the application.
+Create a new file called `MyExampleCrossChainMessenger.sol` in the `teleporter/contracts/src/CrossChainApplications/MyExampleCrossChainMessenger/`
 
 At the top of the file define the Solidity version to work with, and import the necessary types and interfaces.
 
 ```solidity
 pragma solidity 0.8.18;
 
 import {ITeleporterMessenger, TeleporterMessageInput, TeleporterFeeInfo} from "../../Teleporter/ITeleporterMessenger.sol";
+import {ReentrancyGuard} from "@openzeppelin/contracts/security/ReentrancyGuard.sol";
 import {ITeleporterReceiver} from "../../Teleporter/ITeleporterReceiver.sol";
 ```
 
-Next, define the initial empty contract.
+Next, define the initial empty contract. We inherit from `ReentrancyGuard` to prevent reentrancy attacks. We inherit from `ITeleporterReceiver` to allow our contract to receive messages from Teleporter.
```

Comment:
Lets avoid using "we", and instead be more directive. Something like `ReentrancyGuard` is inherited to prevent reentrancy attacks, and `ITeleporterReceiver` is inherited to allow for receiving messages from Teleporter.